src/xr_3da/entry_point.cpp

```diff
@@ -43,12 +43,12 @@ int entry_point(pcstr commandLine)
         shortcuts.Disable();
 #endif
 
-    pcstr fsltx = "-fsltx ";
+    constexpr char fsltx[] = "-fsltx ";
     string_path fsgame = "";
-    if (strstr(commandLine, fsltx))
+    if (pcstr fsltxPos = strstr(commandLine, fsltx); fsltxPos)
```

Comment:
Readability is much worse, while the impact on performance is very low.

This code runs just once in the early start stage. (right after `main`/`WinMain`)

Reply:
Reverted, thanks!